workflow/applications/gfs_cycled.py

```diff
@@ -167,6 +169,8 @@ def get_task_names(self):
             else:
                 hybrid_tasks += ['eobs', 'eupd', 'echgres']
                 hybrid_tasks += ['ediag'] if self.lobsdiag_forenkf else ['eomg']
+            if self.do_jedisnowda:
+                hybrid_tasks += ['esnowanl']
```

Comment:
Just curious if one can run JEDI SNOW DA in the deterministic only, but not in the ensemble when running in hybrid.

Reply:
That's kind of what we are doing here, all this job does is recenter the ensemble to the deterministic, we don't have an EDA for the snow.